### PR TITLE
Create New-bornhom-sandbox

### PR DESCRIPTION
**Pull request by Lezago**
Idea for sandbox for new players

### CODE REVIEW

File: New-bornhom-sandbox [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This is a new sandbox for the new players
+
+Architecture:
+The dimensions of the sandbox are 5 meters by 5 meters.
+
```
Comment (KGunnarsson) [inline New-bornhom-sandbox:5]: @Lezago is this code okey for merging or do you have more things to do.

Comment (Lezago) [reply New-bornhom-sandbox:5]: @KGunnarsson - I would like to crowdsource ideas for toys for the sandbox as well. do you have any ideas what kind of toys we could add before launch?

**Review by Lezago — COMMENTED** [review]
additional comment about sandbox toys

File: New-bornhom-sandbox [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+MateriaL:
+We use fine sand from Bornhom, which is terrible to clean off, but feels nice when playing.
```
Comment (KGunnarsson) [inline New-bornhom-sandbox:7]: @Lezago is this perhaps a better line
```suggestion
We use fine sand from Bornhom, which is easy to clean off, but feels nice when playing.
```

File: New-bornhom-sandbox [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This is a new sandbox for the new players
+
+Architecture:
+The dimensions of the sandbox are 5 meters by 5 meters.
```
Comment (Lezago) [inline New-bornhom-sandbox:4]:
```suggestion: let's change the dimensions to 7 x 7, so there is space for a wooden house.
The dimensions of the sandbox are 7 meters by 7 meters.
```

**Review by Lezago — COMMENTED** [review]
change of dimensions

**Pull request merged**